### PR TITLE
Add quick note

### LINKED ISSUE #200: Feature: Enhance Alarm Dismissal with a Quick Note Feature
### Description: 
Currently, the app lacks a feature that may allow users to set custom quick notes when setting an alarm. Users may want to add personalized notes that appear when the alarm goes off, enhancing their morning routine and providing additional context or reminders.

### Proposed solution: 
Implement a feature that enables users to set custom quick notes while setting an alarm. These notes should be displayed on the alarm dismissal screen, providing users with a personalized message when they wake up or simply, the alarm goes off. The implementation involves:

- Provide with an option (a dialog box) to add a quick note on the "add a new alarm" screen.
- If provided with a quick note, it would show up on the screen when the alarm goes off.
- Else, it just goes the vanilla way, i.e. no quick note, if not provided with one.
- Usage of shared preferences or a local database for storing the quick note.

### Example:
- The user sets a new alarm.
- During the alarm setting process, there is an option to add a custom quick note.
- The user enters a note such as "Have a productive day!" or "Remember to bring lunch." or some other note.
- The note is stored locally and associated with that specific alarm.
- When the alarm goes off, the note is displayed on the alarm dismissal screen.

Please consider assigning this to me if you find this to be acceptable

### PR DESCRIPTION
**Pull request by its-me-abhishek**
### Description
Added quick note facility
## Checklist

- [x] Tests have been added or updated to cover the changes
- [ ] Documentation has been updated to reflect the changes
- [x] Code follows the established coding style guidelines
- [x] All tests are passing

### CODE REVIEW

**Comment by its-me-abhishek** [comment]
Please review my PR @MarkisDev

File: lib/app/modules/about/views/about_view.dart [hunk 1]
```diff
@@ -69,7 +69,9 @@ class AboutView extends GetView<AboutController> {
             Container(
               padding: const EdgeInsets.symmetric(horizontal: 20),
               child: const Text(
-                "This project was originally developed as part of Google Summer of code under the CCExtractor organization. It's free, the source code is available, and we encourage programmers to contribute.",
```
Comment (MarkisDev) [inline lib/app/modules/about/views/about_view.dart:72]: Please revert this change. We have changed the version and text.

File: lib/app/utils/utils.dart [hunk 2]
```diff
@@ -567,100 +566,36 @@ class Utils {
     );
   }
 
-  static Future<void> playCustomSound(String customRingtonePath) async {
```
Comment (MarkisDev) [inline lib/app/utils/utils.dart:570]: Why have you removed the function?

**Review by MarkisDev — CHANGES_REQUESTED** [review]
The about page has to be checked again and any reason why you've changed the ringtone functionality?

File: lib/app/modules/alarmRing/views/alarm_ring_view.dart [hunk 3]
```diff
@@ -151,11 +146,34 @@ class AlarmControlView extends GetView<AlarmControlController> {
                     ],
                   ),
                 ),
+                Obx(
+                  () => Padding(
+                    padding: EdgeInsets.all(0),
+                    child: Column(
+                      children: [
+                        Text(
+                          'Quick Note',
```
Comment (MarkisDev) [inline lib/app/modules/alarmRing/views/alarm_ring_view.dart:155]: Also we should probably just call it note

File: lib/app/modules/alarmRing/views/alarm_ring_view.dart [hunk 4]
```diff
@@ -151,11 +146,34 @@ class AlarmControlView extends GetView<AlarmControlController> {
...
+                          style: Theme.of(context)
+                              .textTheme
+                              .displayLarge!
+                              .copyWith(fontSize: 25),
+                        ),
+                        Text(
+                          controller.currentlyRingingAlarm.value.quickNote,
+                          style: Theme.of(context)
+                              .textTheme
+                              .displayLarge!
```
Comment (MarkisDev) [inline lib/app/modules/alarmRing/views/alarm_ring_view.dart:165]: Maybe use a different font for the note as well

**Comment by its-me-abhishek** [comment]
These might have happened as after syncing with the latest commits that changed the version of the application/redo-ed the custom ringtone, my main branch PR had got a lot of conflict issues. I'll make a new branch and then redo the quick note facility as I did not intend to change or alter the About page or the Ringtone functions.

**Comment by MarkisDev** [comment]
> These might have happened as after syncing with the latest commits that changed the version of the application/redo-ed the custom ringtone, my main branch PR had got a lot of conflict issues. I'll make a new branch and then redo the quick note facility as I did not intend to change or alter the About page or the Ringtone functions.

That's okay, it happens! Remember to never put up a PR which has merge conflicts :)

**Comment by its-me-abhishek** [comment]
Got it! Also, shall I close this PR and put up a newer one?

**Comment by MarkisDev** [comment]
Sure, that should be fine.

**Pull request closed without merging**